feat(apps/whale-api): add updatemasternode indexer (#1869)
DeFiCh/jellyfish: docs/node/CATEGORIES/11-masternode.md

}
```

## updatemasternode

> Creates (and submits to local node and network) a masternode update transaction which update the masternode operator addresses, spending the given inputs..

Reviewer: ```suggestion
Creates a transaction to update masternode operator address, owner address, and/or reward address.
```
